src/main/utils/order-date-utils.ts

```diff
@@ -0,0 +1,14 @@
+import {Task} from '../models/task';
+import { Request} from 'express';
+
+const taskDateOrderUtils = (req: Request): void => {
+  req.session.myTasks = req.session.myTasks.sort((a: Task,b: Task) => a.dateOrder - b.dateOrder);
```

Comment:
Do we really need this line here? looks like you do the same in the line number 7, right?